src/gui/tray/TalkReply.qml

```diff
@@ -0,0 +1,84 @@
+import QtQuick 2.15
+import Style 1.0
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import com.nextcloud.desktopclient 1.0
+
+ColumnLayout {
+    id: talkLayout
```

Comment:
@camilasan I don't remember where I've seen this (probably in Qt docs examples), but, it is better to use `root` as a name of the surrounding component in case of a separate QML file per component.